connect_ext_ppr/tasks_manager.py

```diff
@@ -71,15 +82,38 @@ def _check_cbc_task_status(cbc_service, tracking_id):
     raise TaskException(f'Something went wrong with task: {tracking_id}')
 
 
-def validate_ppr(deployment_request):
+def validate_ppr(deployment_request, **kwargs):
     return True
 
 
-def apply_ppr_and_delegate_to_marketplaces(deployment_request):
+def check_and_update_product(deployment_request, cbc_service, **kwargs):
+    if not deployment_request.manually:
+        deployment = deployment_request.deployment
+        product_id = deployment.product_id
```

Comment:
Tiny comment. Might be written as:
`product_id = deployment_request.deployment.product_id`.
Just an idea.

Reply:
updated!